pcdsdevices/sqr1.py

```diff
@@ -0,0 +1,43 @@
+"""
+Ophyd class for square-one tri-sphere motion system (https://www.sqr-1.com/tsp.html)
+"""
+
+import logging
+
+from ophyd.device import Device
+from ophyd.device import FormattedComponent as FCpt
+from ophyd.pv_positioner import PVPositionerIsClose
+from ophyd.signal import EpicsSignal, EpicsSignalRO
+
+logger = logging.getLogger(__name__)
+
+
+class SQR1Axis(PVPositionerIsClose):
+    setpoint = FCpt(EpicsSignal, '{self.prefix}:TARGET:{self._axis}', kind='normal')
+    readback = FCpt(EpicsSignal, '{self.prefix}:TARGET:{self._axis}:RBV', kind='normal')
```

Comment:
I suspect the readback should be "hinted" here, it's the most important component for these scans. This will make it show up in the automatic table/plotting callbacks during bluesky scans.
```suggestion
    readback = FCpt(EpicsSignal, '{self.prefix}:TARGET:{self._axis}:RBV', kind='hinted')
```